development/components/form/types-reference/unit-type-extension.md

```diff
@@ -18,19 +20,19 @@ None.
 
 ## Code example
 
-Add `TextWithUnitType` to your form and optionally you can configure `unit` for it.
+Add `NumberType` or `IntegerType` to your form and optionally you can configure `unit` for it.
 
 ```php
 <?php
 
 use Symfony\Component\Form\AbstractType;
-use PrestaShopBundle\Form\Admin\Type\TextWithUnitType;
+use PrestaShopBundle\Form\Admin\Type\NumberType;
```

Comment:
I guess L29 must be `use Symfony\Component\Form\Extension\Core\Type\NumberType;`